src/eval/cache/incremental.rs

```diff
@@ -26,8 +33,10 @@ pub struct IncNode {
     bty: BindingType,
     // The state of the node.
     state: IncNodeState,
+    // Forward links to dependencies.
```

Comment:
The documentation for links would be clearer by stating what "backward" and "forward" means in practice. Something like "Nodes that depends on this node" or "Dependencies of this node"